recipe/meta.yaml

```diff
@@ -7,9 +7,12 @@ package:
 source:
   url: http://www.openssl.org/source/openssl-{{ version }}.tar.gz
   sha256: 14c826f07c7e433706fb5c69fa9e25dab95684844b4c962a2cf1bf183eb4690e
+  patches:
+    # https://github.com/openssl/openssl/issues/22811
+    - winsock2.patch
```

Comment:
I normally prefer to put patches into a separate `patches/` folder, but not important here.